packages/gp2-components/src/templates/UsersPage.tsx

```diff
@@ -45,6 +93,23 @@ const UsersPage: React.FC<UsersPageProps> = ({
   const { users } = gp2;
   const backHref = users({}).$;
   const onBackClick = () => changeLocation(backHref);
+
+  const onRemove = useCallback(
+    (value: ValueProps) => {
+      const { id } = value;
+      const [key, val] = id.split('_');
+      const newFilter = (filters[key as keyof FiltersType] || []).filter(
+        (i) => i !== val,
+      );
+      const updatedFilters = {
+        ...filters,
+        [key]: newFilter,
+      };
+      updateFilters(backHref, updatedFilters);
+    },
+    [filters, backHref, updateFilters],
+  );
```

Comment:
I'm not sure I understand this method. 

It's called onRemove but it's setting a filter? Maybe we could have a chat to help me understand it?